docs-site/content/docs/getting-started/guide.md

```diff
@@ -504,9 +504,9 @@ Edit `src/controllers/articles.rs`:
 #![allow(clippy::unused_async)]
 use loco_rs::prelude::*;
 
-use crate::models::_entities::articles;
+use crate::models::_entities::articles::{self, Model};
 
-pub async fn list(State(ctx): State<AppContext>) -> Result<Response> {
```

Comment:
Not sure if this correct or just missing `use axum::response::Response;` at the top?